lib/crc32_power.c

```diff
@@ -120,6 +130,7 @@ __crc32_vpmsum(unsigned int crc, const void* p, unsigned long len) {
 
         vconst1 = vec_ld(offset, vcrc_short_const);
         vdata0 = vec_ld(0, (__vector unsigned long long*) p);
+        VEC_PERM(vdata0, vdata0, vconst1, vperm_const);
```

Comment:
@amock vperm_const is defined only for big-endian. Do we need it for LE, even though VEC_PERM is an empty macro?

Reply:
We do not need it since it vanishes with the empty macro on LE.